specification/translation/Azure.AI.TextTranslation/main.tsp

```diff
@@ -2,6 +2,7 @@ import "@typespec/rest";
 import "@typespec/versioning";
 import "@azure-tools/typespec-azure-core";
 import "./routes.tsp";
+import "./client-names.tsp";
```

Comment:
Do not import client-names.tsp in main.tsp. And rename client-names to client.tsp.

Reply:
With that change, I can see that the custom names were removed, I don't think it is expected:
![image](https://github.com/Azure/azure-rest-api-specs/assets/5716362/4fa41b32-1fb3-427f-8c93-6debe909fc44)

Reply:
Code generation will not use the generated openapi document. You should only use the code generation decorators to control the client library.

Reply:
Changed.